src/tools/perf/perftest_rte.h

```diff
@@ -0,0 +1,33 @@
+/**
+* Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2024. ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#ifndef UCX_PERFTEST_RTE_H
+#define UCX_PERFTEST_RTE_H
+
+#include <ucs/type/status.h>
+
+enum perftest_rte_type {
```

Comment:
maybe `typedef enum {} perftest_rte_type_t;` so that when use it you can skip `enum` keyword

Reply:
fixed